src/test/java/selenium/tutorial/studies/SingleClickNVerifyTest.java

```diff
@@ -17,7 +17,7 @@
 	@Test
 	public void loadBrowser() throws IOException {
 		
-		InputStream input=SampleTest1.class.getClassLoader().getResourceAsStream("testData.properties");
+		InputStream input=SingleClickNVerifyTest.class.getClassLoader().getResourceAsStream("testData.properties");

```

Comment:
are we using this class ? if not remove unnecessary classes

Reply:
Done